client/src/components/DragPanel/index.tsx

```diff
@@ -1,12 +1,24 @@
-import React, { useMemo, memo } from 'react';
 import { Button, Spinner } from '@blueprintjs/core';
+import React, { memo } from 'react';
```

Comment:
Please, do not shuffle imports, its ok to sort them by `builtin`, `3rd party`, `user defined`, but there is a way to do it much more cleaner by sorting them using following rules:
* our (user defined) modules can be grouped in:
  - components
  - domain-related
  - another group (?)
  - etc
 * insert empty line between groups